checkstyle_suppressions.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0"?>
+
+<!DOCTYPE suppressions PUBLIC
+        "-//Puppy Crawl//DTD Suppressions 1.2//EN"
+        "http://www.puppycrawl.com/dtds/suppressions_1_2.dtd">
+
+<suppressions>
+    <suppress checks="Javadoc" files="."/>
```

Comment:
Is this best practice to disable Javadoc warnings?

Reply:
Happy to remove. But we would have to add what is document in the below link to all our java files. 
https://checkstyle.sourceforge.io/config_javadoc.html